platform/android/java/app/build.gradle

```diff
@@ -124,6 +124,12 @@ android {
             // - https://stackoverflow.com/a/44704840
             useLegacyPackaging shouldUseLegacyPackaging()
         }
+
+        // Always select Godot's version of libc++_shared.so in case deps have their own
+        pickFirst 'lib/x86/libc++_shared.so'
+        pickFirst 'lib/x86_64/libc++_shared.so'
+        pickFirst 'lib/armeabi-v7a/libc++_shared.so'
+        pickFirst 'lib/arm64-v8a/libc++_shared.so'
```

Comment:
I'm assuming there's gradle logic to still handle the arch properly only "pickFirst" the one that actually matches the target arch?

Reply:
yes, this specifies a pattern and so it picks the ones that match the given patterns.